### DISCUSSION TITLE
Getting the exact time of the start of the experiment

### DISCUSSION THREAD

**Discussion by Quinlanptq** [jspsych/jsPsych#3250] [General]
I see that there is a trial variable called time_elapsed that registers the number of miliseconds from the start of the experiment. I wondered if it is possible to recover the actual start time of the experiment which counts as the zero point for time_elapsed?

Thanks in anticipation,

Philip.

**Comment by Quinlanptq** [comment]
Sorry please ignore 

check out 

 jsPsych.getStartTime()

**Comment by Quinlanptq** [comment]
Perhaps another silly question but it keeps telling me that in this code (7.3)

var starte = jsPsych.getStartTime();

jsPsych.getStartTime() is not a function

**Comment by Shaobin-Jiang** [comment]
Are you still using jsPsych v6? `getStartTime` is only available in jsPsych v7.
